checks/devnet/devnet-data-continue.json

```diff
@@ -19,7 +19,7 @@
   "data": {
     "active_reconciliation_concurrency": 64,
     "inactive_reconciliation_concurrency": 32,
-    "inactive_reconciliation_frequency": 1,
+    "inactive_reconciliation_frequency": 16,
```

Comment:
This is the number of blocks to wait between inactive reconciliations on each account.